### DISCUSSION TITLE
[Doc error]  Tags for createDraftListing / updateListing: string[] vs string

### DISCUSSION THREAD

**Discussion by kapik** [etsy/open-api#780] [Q&A]
Hello 👋

There is a confusion in the api documentation for the `createDraftListing` and `updateListing` methods.
The `tags` property must be a `string` (each tag separated by a comma) and not an `Array of strings`.

🚀

**Comment by mjec** [comment]
Hi @kapik!

This is in fact intentional and [documented](https://developers.etsy.com/documentation/essentials/urlsyntax#parameter-types).

As you'd be aware, the `application/x-www-form-urlencoded` MIME type (used for submitting data to the Etsy API) does not support types for any data; it can only send a sequence of string keys and associated string values. When we want to accept an array of values, that array must be somehow converted into a string. The serialization mechanism we support is joining the contents of the array with `,` characters. This works for strings in the `tags` parameter, but also integers for `image_ids` etc.
